content/en/docs/concepts/configuration/scheduler-perf-tuning.md

```diff
@@ -31,12 +31,14 @@ once it finds a certain number of them. This improves the scheduler's
 performance in large clusters. The number is specified as a percentage of the
 cluster size. The percentage can be controlled by a configuration option called
 `percentageOfNodesToScore`. The range should be between 1 and 100. Larger values
-are considered as 100%. Kubernetes 1.14 has logic to find the percentage of
-nodes to score based on the size of the cluster if it is not specified in the
-configuration. It uses a linear formula which yields 50% for a 100-node cluster.
-The formula yields 10% for a 5000-node cluster. The lower bound is 5%. In other
-words, the scheduler will always scores at least 5% of the cluster no matter how
-large the cluster is.
+are considered as 100%. Zero is equivalent to not providing the config option.
+Kubernetes 1.14 has logic to find the percentage of nodes to score based on the
+size of the cluster if it is not specified in the configuration. It uses a
+linear formula which yields 50% for a 100-node cluster. The formula yields 10%
```

Comment:
So I can tell that you have completely captured the behavior with this description, but most readers probably won't want to solve linear equations to parse this. How about we give either a table with a few extra data points, or add a chart?

Reply:
I believe most users will be happy with the default behavior and may not care about the actual value for their clusters. Dynamic changes of the cluster size, for example due to autoscaling, make this value variable by time. So, I am not sure if providing a table for various cluster sizes is very valuable. Most users should not ever need to think about this option.